source/_includes/documentation-shell-integration.md

```diff
@@ -5,12 +5,12 @@ The easiest way to install shell integration is to select the *iTerm2&gt;Install
 
 When you select the *iTerm2&gt;Install Shell Integration* menu item, it types this for you:
 
-<pre>curl -L https://iterm2.com/misc/install_shell_integration.sh | bash</pre>
+<pre>curl -SsL https://iterm2.com/misc/install_shell_integration.sh | bash</pre>
```

Comment:
We send `curl -L`. Should we use -Ss for some reason? Seems useful to get feedback that something is happening.